About.md

```diff
@@ -0,0 +1,63 @@
+# About the Lattice Standard
+
+## What the Lattice Standard is
+
+1. The Lattice Standard is an *abstract* description of a lattice for the simulated transport of charged particles.
+
+1. A Lattice is an ordered list of regions in space distinguished by the presense of (possibly time-varying) specially described electromagnetic fields,
+materials, apertures and other possible engineered structures. For simplicity, we will denote these regions as _lattice_elements_ although a region
```

Comment:
Instead of "For simplicity, we will denote these regions as _lattice_elements_ " better I think is "These regions will be called _lattice elements_

Reply:
Sounds good to me.

Reply:
Point 2 and 4 seem too specific to me in this context, as they imply a specific choice of the relation between lattice, elements and positioning. I would state for instance that “the standard will define what is a lattice and how it can be described in terms of primitive components".

For instance, you can call a "lattice element" a coordinate transformation, observation point, a matrix element that are not regions of space, but which you could order. Conversely, you may call elements, specifications of certain aspects of a region of space (fields, aperture, etc,), without an explicit positioning and the lattice a unordered collection of the positioning of existing elements relative to a reference trajectory.